Source/Common/IGListIndexPathResult.m

```diff
@@ -22,12 +22,12 @@ - (instancetype)initWithInserts:(NSArray<NSIndexPath *> *)inserts
                 oldIndexPathMap:(NSMapTable<id<NSObject>, NSIndexPath *> *)oldIndexPathMap
                 newIndexPathMap:(NSMapTable<id<NSObject>, NSIndexPath *> *)newIndexPathMap {
     if (self = [super init]) {
-        _inserts = [inserts copy];
-        _deletes = [deletes copy];
-        _updates = [updates copy];
-        _moves = [moves copy];
-        _oldIndexPathMap = [oldIndexPathMap copy];
-        _newIndexPathMap = [newIndexPathMap copy];
+        _inserts = inserts;
+        _deletes = deletes;
+        _updates = updates;
+        _moves = moves;
+        _oldIndexPathMap = oldIndexPathMap;
+        _newIndexPathMap = newIndexPathMap;
```

Comment:
Normally I'd say this is good practice, but since this is part of the private API we're fine I think